scripts/install/functions.sh

```diff
@@ -85,43 +85,44 @@ function create_shell_app {
 
 function add_b2b {
     if [ "${ADD_B2B_LIBS}" = true ] ; then
-        ng add --skip-confirmation @spartacus/organization@${SPARTACUS_VERSION} --interactive false
-        ng add --skip-confirmation @spartacus/checkout@${SPARTACUS_VERSION} --interactive false --features="Checkout-B2B" --features="Checkout-Scheduled-Replenishment"
+        ng add @spartacus/organization@${SPARTACUS_VERSION} --skip-confirmation --no-interactive
+        ng add @spartacus/checkout --skip-confirmation --no-interactive --features="Checkout-B2B" --features="Checkout-Scheduled-Replenishment"
```

Comment:
General comment for PR:  The SSR starts, but the CSR does not start.
PM2 says 
```
Error [ERR_REQUIRE_ESM]: require() of ES Module .../.nvm/versions/node/v16.14.2/lib/node_modules/serve/build/main.js not supported.
main.js is treated as an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which declares all .js files in that package scope as ES modules.
Instead rename main.js to end in .cjs, change the requiring code to use dynamic import() which is available in all CommonJS modules, or change "type": "module" to "type": "commonjs" in .../.nvm/versions/node/v16.14.2/lib/node_modules/serve/package.json to treat all .js files as CommonJS (using .mjs for all ES modules instead).

    at Object.<anonymous> (.../.nvm/versions/node/v16.14.2/lib/node_modules/pm2/lib/ProcessContainerFork.js:33:23) {
  code: 'ERR_REQUIRE_ESM'
}
```

Reply:
This error still happens.  Is it caused by changes from this PR?

Reply:
In the en this is a know issue and it is unrelated to the changes in the PR.
https://jira.tools.sap/browse/CXSPA-788